lib/active_fulfillment/services/shopify_api.rb

```diff
@@ -86,11 +88,22 @@ def send_app_request(action, headers, data)
         end
       end
 
-      log :info, "GET response action=#{action} in #{"%.4fs" % realtime} #{response}"
-
+      log_response(action, realtime, response)
       response
     end
 
+    def log_response(action, realtime, response)
+      if response.try(:length).to_i > MAX_LOGGED_RESPONSE_LENGTH
+        truncated = response.to_s[0..MAX_LOGGED_RESPONSE_LENGTH]
```

Comment:
`.truncate(` is not defined for some of the CI tests

Reply:
Oh, version thing?

Reply:
Oh yeah we've got some ancient versions of ruby and rails in the travis file. Many end of life.